modules/bitgo/test/browser/browser.spec.ts

```diff
@@ -19,6 +19,7 @@ describe('Coins', () => {
             'EosToken': 1,
             'AlgoToken': 1,
             'AvaxCToken': 1,
+            'PolygonToken': 1,
```

Comment:
Browser tests were failing if I didn't include PolygonToken here 
I see we have includes other tokens as well